setup.py

```diff
@@ -12,7 +12,7 @@
 from setuptools import setup, find_packages, Extension
 from setuptools.command.build_ext import build_ext
 
-VERSION = "3.0.1"
+VERSION = "3.x.x-dev"
```

Comment:
```suggestion
# The VERSION line is edited automatically during deployments.
# You may change the contents of the string, but do not otherwise edit the line.
VERSION = "3.x.x-dev"
```

Reply:
Done